jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -148,13 +168,16 @@ public Configuration(
     this.reporterConfig = reporterConfig;
 
     statsFactory = new StatsFactoryImpl(new NullStatsReporter());
+
+    this.disableGlobalTracer = disableGlobalTracer;
   }
 
   public static Configuration fromEnv() {
     return new Configuration(
         getProperty(JAEGER_SERVICE_NAME),
         SamplerConfiguration.fromEnv(),
-        ReporterConfiguration.fromEnv());
+        ReporterConfiguration.fromEnv(),
+        Boolean.valueOf(getProperty(JAEGER_DISABLE_GLOBAL_TRACER)));
```

Comment:
use getPropertyAsBoolean

Reply:
`getPropertyAsBoolean` returns null when the property isn't set, which isn't what we want here.